libs/model/src/community/CreateGroup.command.ts

```diff
@@ -7,7 +7,8 @@ import { mustBeAuthorized, mustNotExist } from '../middleware/guards';
 import { GroupAttributes } from '../models';
 
 export const MAX_GROUPS_PER_COMMUNITY = 20;
-export const Errors = {
+
+export const CreateGroupErrors = {
   MaxGroups: 'Exceeded max number of groups',
   InvalidTopics: 'Invalid topics',
 };
```

Comment:
Curious– are we currently able to share error constants with the frontend?

Reply:
Don't think so, we'll have to move all error constants to libs/schemas if we want to